BREAKING-CHANGES.md

```diff
@@ -136,8 +143,6 @@ This change simplifies the trait and makes it easier to implement.
 
 ### `Frame::size` is deprecated and renamed to `Frame::area`
 
-[#1293]: https://github.com/ratatui/ratatui/pull/1293
```

Comment:
Is this change related?

Reply:
This is just markdownlint removing unused link refs - I fixed this in another PR IIRC.

Reply:
https://github.com/ratatui/ratatui/pull/1373#discussion_r1760509268